test/mean-module.test.js

```diff
@@ -71,7 +73,11 @@ describe('Module Sub Generators Tests', function () {
       assert.file('modules/foo/client/services/');
       assert.file('modules/foo/client/views/');
       assert.file('modules/foo/tests/client/');
-      assert.file('modules/foo/server/config');
+      assert.file('modules/foo/server/config/');
+      assert.file('modules/foo/server/controllers/');
+      assert.file('modules/foo/server/models/');
+      assert.file('modules/foo/server/policies/');
+      assert.file('modules/foo/server/routes/');
       assert.file('modules/foo/tests/server/');
     });
```

Comment:
@UndeadBaneGitHub How did these tests pass through Travis CI? Does the code that I pointed out, work in some environments, but not in others?

https://github.com/meanjs/generator-meanjs/pull/190/files#diff-4d48c9fe091089030f87da7dda259512R157